rest-service/manager_rest/resources_v2.py

```diff
@@ -21,12 +21,14 @@
 from datetime import datetime
 from collections import OrderedDict
 
-from flask_securest.rest_security import SecuredResource
 
+import elasticsearch
 from flask_restful_swagger import swagger
 from flask import request
 from flask.ext.restful import marshal
 
+from flask_securest.rest_security import SecuredResource
```

Comment:
why blank lines both above and below? either we treat it as a 3rd party or as a cloudify library, no?